absl/container/CMakeLists.txt

```diff
@@ -47,10 +47,16 @@ list(APPEND CONTAINER_INTERNAL_HEADERS
   "internal/unordered_set_modifiers_test.h"
 )
 
+# container library
+list(APPEND CONTAINER_SRC
+  "internal/raw_hash_set.cc"
+)
 
-absl_header_library(
+absl_library(
   TARGET
     absl_container
+  SOURCES
+    ${CONTAINER_SRC}
```

Comment:
Please directly add `"internal/raw_hash_set.cc"` here instead of creating CMake global variable `CONTAINER_SRC`. Same goes to other CMake variables.

See #113

Reply:
Everything inlined now.